includes/class-wc-calypso-bridge-woocommerce-admin.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Contains customizations for WooCommerce Admin
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.0.17
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge WooCommerce Admin
+ */
+class WC_Calypso_Bridge_WooCommerce_Admin {
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_WooCommerce_Admin instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function factory() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Initialize the class
+	 */
+	public function init() {
+		if ( ! apply_filters( 'wc_calypso_bridge_allow_wca_devdocs', false ) ) {
```

Comment:
Optional: In https://github.com/Automattic/wc-calypso-bridge/pull/479 I added Jetpack-similar style pattern for development flag/filter using `wc_calypso_bridge_development_mode`.

We could optionally use the same filter here as well so we don't have to add a bunch of different flags to get a dev environment set up.

Reply:
filter updated in https://github.com/Automattic/wc-calypso-bridge/pull/477/commits/eed62b5737e29a26e3992eca78655a3961258183